axelrod/player.py

```diff
@@ -28,14 +28,22 @@ class Player(object):
     """
 
     name = "Player"
+    behaviour = {}
 
     def __init__(self):
         """Initiates an empty history and 0 score for a player."""
         self.history = []
-        self.stochastic = "random" in inspect.getsource(self.__class__)
-        self.tournament_attributes = {'length': -1, 'game': None}
+        self.behaviour['stochastic'] = (
+            "random" in inspect.getsource(self.__class__))
         if self.name == "Player":
-            self.stochastic = False
+            self.behaviour['stochastic'] = False
+        if 'memory_depth' not in self.behaviour:
```

Comment:
> These could all be kept in a default_behaviour dictionary and this could be replaced with a loop over the keys of it yes?

Yes, that would be neater. I'll sort that.